docs/api-reference/engine/geometries.md

```diff
@@ -0,0 +1,112 @@
+# Built-in Geometries
+
+**@luma.gl/engine** provides several built in geometry primitives (subclasses of [Geometry](/docs/api-reference/engine/geometry.md)). The generated geometry instances will have `indices` and `POSITION`, `NORMAL` and `TEXCOORD_0` attributes.
+
+## ConeGeometry
```

Comment:
Maybe just move to math.gl - and if so, recommend keeping these as one doc file per class as is the convention in that repo.